docs/src/intro-java.md

```diff
@@ -1,150 +1,96 @@
 ---
 id: intro
-title: "Getting started"
+title: "Installation"
 ---
 
-<!-- TOC -->
-- [Release notes](./release-notes.md)
-
-## Installation
+Playwright was created specifically to accommodate the needs of end-to-end testing. Playwright supports all modern rendering engines including Chromium, WebKit, and Firefox. Test on Windows, Linux, and macOS, locally or on CI, headless or headed with native mobile emulation. 
 
 Playwright is distributed as a set of [Maven](https://maven.apache.org/what-is-maven.html) modules. The easiest way to use it is to add one dependency to your project's `pom.xml` as described below. If you're not familiar with Maven please refer to its [documentation](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html).
 
-## Usage
+Get started by installing Playwright and running the example test to see it in action.
 
-<Tabs
-  defaultValue="java"
-  values={[
-    {label: 'Example.java', value: 'java'},
-    {label: 'pom.xml', value: 'pom'}
-  ]
-}>
-<TabItem value="java">
+Create a new [Maven](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html) project:
 
-```java
-package org.example;
+```bash
+mvn archetype:generate -DgroupId=com.mycompany.app -DartifactId=my-app -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false
+```
 
-import com.microsoft.playwright.*;
+Add the Playwright dependency in the `pom.xml` file:
 
-public class Example {
-  public static void main(String[] args) {
-    try (Playwright playwright = Playwright.create()) {
-      Browser browser = playwright.chromium().launch();
-      Page page = browser.newPage();
-      page.navigate("http://playwright.dev");
-      System.out.println(page.title());
-    }
-  }
-}
+```xml
+<dependency>
+    <groupId>com.microsoft.playwright</groupId>
+    <artifactId>playwright</artifactId>
+    <version>1.27.0</version>
+</dependency>
 ```
 
-</TabItem>
-<TabItem value="pom">
+And set the `<maven.compiler.source>` and `<maven.compiler.target>` to `1.8` in the `pom.xml` file:
```

Comment:
I don't think it it actually required, it is supposed to work well with a newer versions. It might be a requirement of the maven plugin, needs validation.